src/Microsoft.AspNetCore.Razor.LanguageServer/RazorCompletionEndpoint.cs

```diff
@@ -107,13 +115,19 @@ public async Task<CompletionList> Handle(CompletionParams request, CancellationT
                     Detail = razorCompletionItem.Description,
                     Documentation = razorCompletionItem.Description,
                     FilterText = razorCompletionItem.DisplayText,
-                    SortText = razorCompletionItem.DisplayText,
+                    SortText = "__Razor__" + razorCompletionItem.DisplayText,
```

Comment:
What is this for?

Reply:
When VSCode has all the completion items it controls what order they show up in alphabetically. Means you could have `@page` show up first even though it begins with p if it's sort text starts with an A. In this case I shouldn't be changing the sort text. Will undo